packages/cli/templates/open-next.config.ts

```diff
@@ -0,0 +1,8 @@
+// @ts-nocheck
+// default open-next.config.ts file created by @opennextjs/cloudflare
+import { defineCloudflareConfig } from "@opennextjs/cloudflare/config";
+import r2IncrementalCache from "@opennextjs/cloudflare/overrides/incremental-cache/r2-incremental-cache";
+
+export default defineCloudflareConfig({
+	incrementalCache: r2IncrementalCache,
```

Comment:
@chanceaclark I can't remember, did we change this to `kvIncrementalCache` just for experimentation? Or should I also change it here?